www/src/components/guidelines/color/modal.js

```diff
@@ -196,19 +229,34 @@ const ColorModal = ({ palette, color, handleModalClose }) => {
 
   return (
     <>
-      <Flex alignItems="baseline" p={{ xxs: 6, md: 8 }}>
+      <Flex
+        sx={{
+          alignItems: `baseline`,
+          p: 6,
+          [mediaQueries.md]: {
```

Comment:
Again, I think these are ideal for `theme-ui`s "responsive array" syntax. You can omit values for breakpoints via `null`, see e. g. https://github.com/gatsbyjs/gatsby/blob/79986bdf4da863dea49d20ec69098f6dc0f56fcd/www/src/components/navigation.js#L160